<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="PF35M749" failures="0" tests="5" name="LoginTest Error" time="1.51" errors="0" timestamp="2023-03-29T18:30:16 CEST">
  <testcase classname="TestCases.TestCollection" name="loginWithUsernameAndPassword" time="0.32"/>
  <testcase classname="TestCases.TestCollection" name="loginWithUsernameAndPassword" time="0.201"/>
  <testcase classname="TestCases.TestCollection" name="loginWithUsernameAndPassword" time="0.19"/>
  <testcase classname="TestCases.TestCollection" name="loginWithUsernameAndPassword" time="0.206"/>
  <testcase classname="TestCases.TestCollection" name="loginWithUsernameAndPassword" time="0.29"/>
  <testcase classname="TestCases.TestCollection" name="createNewConnectionAndValidateStats" time="0.0">
    <ignored/>
  </testcase> <!-- createNewConnectionAndValidateStats -->
  <testcase classname="TestCases.TestCollection" name="javaScriptTest" time="0.0">
    <ignored/>
  </testcase> <!-- javaScriptTest -->
  <testcase classname="TestCases.TestCollection" name="expectedConditionTest" time="0.0">
    <ignored/>
  </testcase> <!-- expectedConditionTest -->
  <testcase classname="TestCases.TestCollection" name="loginWithCorrectUsernameAndPassword" time="0.0">
    <ignored/>
  </testcase> <!-- loginWithCorrectUsernameAndPassword -->
  <testcase classname="TestCases.TestCollection" name="createNewConnection" time="0.0">
    <ignored/>
  </testcase> <!-- createNewConnection -->
</testsuite> <!-- LoginTest Error -->
